ocaml/testsuite/tests/parsetree/test.ml

```diff
@@ -170,7 +170,7 @@ let check_all_ast_attributes_and_extensions_start_with raw_parsetree_str ~prefix
 
 let () =
   process "source.ml";
-  Language_extension.enable_maximal ();
+  Language_extension.set_universe_and_enable_all Alpha;
```

Comment:
In these changes, I find the old code easier to understand. Maybe make a synonym `maximal` for `Alpha`. Then this will be easier to understand as just being the maximal set, not something special for Alpha extensions.